File: src/main/java/fi/finlit/edith/ui/services/ServiceException.java
package fi.finlit.edith.ui.services;

public class ServiceException extends RuntimeException{

    private static final long serialVersionUID = -5426150520106835552L;

    public ServiceException(String msg){
        super(msg);
    }

    public ServiceException(Throwable t) {
        super(t);
    }

    public ServiceException(String msg, Throwable t) {
        super(msg,t);
    }

}

File: src/main/scala/fi/finlit/edith/ui/services/ServiceException.scala
package fi.finlit.edith.ui.services

//remove if not needed
import scala.collection.JavaConversions._

@SerialVersionUID(-5426150520106835552L)
class ServiceException(msg: String) extends RuntimeException(msg) {

  def this(t: Throwable) {
    super(t)
  }

  def this(msg: String, t: Throwable) {
    super(msg, t)
  }
}
